.gitignore

```diff
@@ -111,4 +111,17 @@ packages/.DS_Store
 dist/
 test/.DS_Store
 
-.yarn/
\ No newline at end of file
+.yarn/
+
+# Ignore all .properties, .lock, .bin, and other build output files inside the .gradle directory
+**/android/gradle/**/*.properties
```

Comment:
Let's keep the gradle property file.  https://github.com/adobe/aepsdk-react-native/blob/main/apps/AEPSampleApp/android/gradle.properties It includes important Android settings.